sites/docs/src/content/docs/tutorials/google_colab/nf-core-colab-guide.md

```diff
@@ -0,0 +1,178 @@
+
+...and you're ready to go!
+
+Yes, the setup is that simple!
+
+Now that the main parts of the environment are all set up, we can run the demo pipeline [nf-core/demo](https://nf-co.re/demo/).
+To install the pipeline, run:
+
+```python
+! nextflow pull nf-core/demo
+```
+
+Now you can finally run your pipeline!
+
+```python
+! nextflow run nf-core/demo -profile conda,test --outdir demo-results
+```
+
+## Running and Editing Pipelines in VS Code via Colab
+
+While you could get away with editing existing pipelines inside Colab's built-in terminal using editors like vim or nano, a VS Code IDE offers a more robust and richer environment for development.
+
+If you use VS Code as your normal IDE, thankfully the [vscode-colab](https://github.com/EssenceSentry/vscode-colab) Python library provides just the toolkit you need to take advantage of Colab's hardware in the comfort of the popular VSCode software suite.
+
+It allows you to securely and reliably connect a local instance of VS Code to Google Colab as well as Kaggle notebooks through the use of the official [VS Code Remote Tunnels](https://code.visualstudio.com/docs/remote/tunnels) package.
+
+You can read more about the library and even help contribute to new features on its [GitHub repository](https://github.com/EssenceSentry/vscode-colab).
+
+:::note
+If you decide to use this approach, it is best to run this immediately after setting up the other dependencies that require installation by running inside Colab cells to avoid any connection issues.
+:::
+
+The first step is to install and import the library, which can easily be done by running the following command in your Colab code cell:
+
+```python title="Install vscode-colab"
+!pip install vscode-colab
+import vscode_colab
+```
+
+Next, authenticate the connection using your GitHub credentials:
+
+```python
+vscode_colab.login()
+```
+
+![interactive login popup in cell output](/images/tutorials/google_colab/login.png)
+
+Follow the displayed instructions to authorize the connection.
+
+To start the VS Code tunnel, optionally configure Git, set up a Python version, or create a new project:
+
+```python
+vscode_colab.connect(
+    name="my-tunnel",
+    git_user_name="Your Name",
+    git_user_email="you@example.com",
+    setup_python_version="3.13",  # Optional: Specify Python version to install with pyenv
+    create_new_project="my_new_project"  # Optional: Create a new project directory
+)
+```
+
+![interactive tunnel connection popup in cell output](/images/tutorials/google_colab/connect.png)
+
+The above step allows for a lot more customization than shown here. Please refer to the library's [vignette](https://github.com/EssenceSentry/vscode-colab) to see if any of these options will suit your needs.
+
+Then, in your local VS Code:
+
+1. Ensure the [Remote Tunnels extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.remote-server) is installed.
+2. Sign in with the same GitHub account used in the notebook.
```

Comment:
I think, this requires more clarification. Instead of promoting contributing to the library, stress that we are now leaving the browser and opening the code editor as the new primary interface for interacting with the Colab notebook.